python/batch-predictor/requirements.txt

```diff
@@ -1,6 +1,7 @@
-pyspark==2.4.5
+findspark
```

Comment:
What is `findspark` used for?

Reply:
this is for this line which seems to use it in some cases when for some reason spark cannot be found (maybe due to package upgrades etc) https://github.com/gojek/merlin/blob/87dc891183b2eb9d728e70bb569ebba053c64f89/python/batch-predictor/main.py#L34